ios/BannerView.swift

```diff
@@ -69,25 +77,33 @@ class BannerView: UIView, GADAppEventDelegate, GADBannerViewDelegate, GADAdSizeD
     }
 
     func createAdView(){
-        bannerView = DFPBannerView.init()
-
+        if(isFluid){
+            bannerView = DFPBannerView(adSize: kGADAdSizeFluid)
+        }
+        else{
+            bannerView = DFPBannerView.init()
+        }
+        
         let rootViewController = UIApplication.shared.delegate?.window??.rootViewController
-
+        
         var validAdSizes = [NSValue]()
-
-        let adSizesArray = adSizes ?? []
-
-        for sizes in adSizesArray {
+        
+        for sizes in adSizes! {
             let width = sizes[0]
             let height = sizes[1]
             let customGADAdSize = GADAdSizeFromCGSize(CGSize(width: width, height: height))
             validAdSizes.append(NSValueFromGADAdSize(customGADAdSize))
         }
 
+        if(isFluid){
+            validAdSizes.append(NSValueFromGADAdSize(kGADAdSizeFluid))
+        } else{
+            bannerView?.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
I know nothing about Swift, it's cool to see that it also has optional chaining 👌